frontend/src/design/components/SearchInput.js

```diff
@@ -25,7 +25,7 @@ function SearchInput(props) {
             fullWidth
             onChange={onChange}
             onKeyUp={onKeyUp}
-            placeholder="Search"
+            placeholder="SearchIcon"
```

Comment:
Was it changed intentionally? Looks like it changed with the renaming.

Reply:
Ok, it seems not just text but the import of the icon. If so, please disregard the comment above

Reply:
Its description from the library says: `The short hint displayed in the input before the user enters a value` so I think you are right, it should be reverted. Probably got updated by mistake with a find and replace or something.

Reply:
If it's not importing, but just a placeholder then let's revert it for every change pair "Search" -> "SearchIcon" (I have come across this change in many places in the PR)

Reply:
Ok, will update that

Reply:
I reverted all the changes for the placeholders